File: 02 Digits and Numbers - Foundation Programs/program-04.cpp
// Write a program that takes a 3-digit number as input and extracts and displays the middle digit.

#include<iostream>
using namespace std;
int main()
{
    int number,middle_digit;
    cout<<"Enter 3-digit number : ";
    cin>>number;
    
    middle_digit=(number/10)%10;

    cout<<"The Middle Digit is : "<<middle_digit;

    return 0;
}

File: archit_vishwakarma/Level-1/03-Digits-And-Numbers/program-04.py
n = input("Enter a number : ")

try:
    n = int(n)
    if n < 1000 and n > 99:
        last_2 = n%100
        mid = last_2//10
        print(f"middle digit : {mid}")
    else:
        print("Enter a Three Digit Number")
except ValueError:
    print("enter a valid number")